packages/fx-core/tests/core/middleware/testAssets/v3Migration/happyPath/testCaseFiles/.env.dev

```diff
@@ -0,0 +1,33 @@
+STATE__SOLUTION__PROVISIONSUCCEEDED=true
+STATE__SOLUTION__TEAMSAPPTENANTID=teamsAppTenantId
+AZURE_SUBSCRIPTION_ID=subscriptionId
+STATE__SOLUTION__SUBSCRIPTIONNAME=subscriptionName
+STATE__SOLUTION__TENANTID=tenantId
+STATE__SOLUTION__NEEDCREATERESOURCEGROUP=true
+AZURE_RESOURCE_GROUP_NAME=resourceGroupName
+STATE__SOLUTION__LOCATION=East US
+RESOURCE_SUFFIX=abcde
+TEAMS_APP_TENANT_ID=tenantId
+TEAMS_APP_ID=teamsAppId
+AAD_APP_CLIENT_ID=clientId
+SECRET_AAD_APP_CLIENT_SECRET={{fx-resource-aad-app-for-teams.clientSecret}}
```

Comment:
one reminder: need to replace this value when migrate userdata, can see how to handle this case later.